### PR TITLE
Main role use sementic tag <main>

### PR DESCRIPTION
**Pull request by sudhir-jedhe**
The role of `main` uses the `<main>` tag without a role attribute

### CODE REVIEW

File: src/widgets/AppWindow.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ export default class AppWindow extends createjs.Container {
       parent: this,
       role: AccessibilityModule.ROLES.MAIN,
     });
-
+    this.accessible.addChild(this._contentArea);
```
Comment (mnewcomb314) [inline src/widgets/AppWindow.js]: Why is this line needed?  I would have expected the object passed to `AccessibilityModule.register` for `this._contentArea` including `parent: this,` would handle this.

Comment (sudhir-jedhe) [reply src/widgets/AppWindow.js]: Yes. i forgot that.

**Comment by mnewcomb314** [comment]
Since the work was to update the main role to use a different tag (which is handled inside of CAM) and there are no code changes needed on the test app side for this due to it already registering a DisplayObject with that role, can the changes for this PR please be only updating package.json to point to CAM 0.3.1 (I'll take care of creating that release once your CAM main role PR merges)?

**Comment by sudhir-jedhe** [comment]
updated

**Review by mnewcomb314 — APPROVED** [review]
